src/pages/Home/Home.page.jsx

```diff
@@ -1,39 +1,32 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
-
-import { useAuth } from '../../providers/Auth';
+import React, { useState } from 'react';
+import { ThemeProvider } from 'styled-components';
+import Navbar from '../../components/Navbar';
+import { GlobalStyles } from '../../components/globalStyles';
+import { lightTheme, darkTheme } from '../../components/Themes';
+import Card from '../../components/Card';
+import { youtube } from '../../utils/youtube-videos-mock';
 import './Home.styles.css';
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+function Home() {
+  const [darkMode, setDarkMode] = useState(true);
+  const cardList = youtube.items.map((video) => (
+    <Card
+      key={video.etag}
+      image={video.snippet.thumbnails.high.url}
+      title={video.snippet.title}
+      description={video.snippet.description}
+    />
+  ));
 
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
-    </section>
+    <>
+      <ThemeProvider theme={darkMode ? darkTheme : lightTheme}>
+        <GlobalStyles />
+        <Navbar darkMode={darkMode} setDarkMode={setDarkMode} />
+        <div className="container-list">{cardList.splice(1)}</div>
```

Comment:
Great, a more common way to loop through a list of items and create several components is something like:

```jsx
{youtube.items.map(video => (
   <Card ... />
))}
```

which is more or less what you did, the difference is that the result of the `map` is not assigned to a variable